### ISSUE TITLE
WebUI: Direkte Verküpfung "Could not open interface file"

### ISSUE THREAD

**Issue by beye91** [jens-maus/RaspberryMatic#549]
Hallo Jens, 
nach dem Update auf  **3.41.11.20190126** , kann ich keine "Direkten Verknüpfungen" mehr erstellen. 

Folgender Fehler erscheint:
![image](https://user-images.githubusercontent.com/35919167/52083968-3c1ce600-25a1-11e9-8888-d5dd790d3c6a.png)

**Comment by jens-maus** [comment]
Dann stimmt hier etwas grundsätzlich nicht denn das "Interface" file sollte definitiv da sein. Bitte mal auf die CCU/RaspberryMatic schauen ob es da eine Datei `/etc/config/InterfacesList.xml` gibt.

**Comment by beye91** [comment]
Nabend Jens, 

# ls /etc/config/InterfacesList.xml
ls: /etc/config/InterfacesList.xml: No such file or directory

Was kann man dagegen tun? Von einer anderen Installation kopieren?

**Comment by jens-maus** [comment]
Existiert das Problem noch?

**Comment by beye91** [comment]
Ja, aktuell noch. Könnem wir uns ja nächste Woche drüber unterhalten. ;)Sent from my Samsung Galaxy smartphone.
-------- Original message --------From: Jens Maus <notifications@github.com> Date: 3/29/19  17:04  (GMT+01:00) To: jens-maus/RaspberryMatic <RaspberryMatic@noreply.github.com> Cc: beye91 <christopher@beye-networks.de>, Author <author@noreply.github.com> Subject: Re: [jens-maus/RaspberryMatic] WebUI: Direkte Verküpfung "Could not open interface file" (#549) Existiert das Problem noch?

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/jens-maus/RaspberryMatic","title":"jens-maus/RaspberryMatic","subtitle":"GitHub repository","main_image_url":"https://github.githubassets.com/images/email/message_cards/header.png","avatar_image_url":"https://github.githubassets.com/images/email/message_cards/avatar.png","action":{"name":"Open in GitHub","url":"https://github.com/jens-maus/RaspberryMatic"}},"updates":{"snippets":[{"icon":"PERSON","message":"@jens-maus in #549: Existiert das Problem noch?"}],"action":{"name":"View Issue","url":"https://github.com/jens-maus/RaspberryMatic/issues/549#issuecomment-478054423"}}}
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/jens-maus/RaspberryMatic/issues/549#issuecomment-478054423",
"url": "https://github.com/jens-maus/RaspberryMatic/issues/549#issuecomment-478054423",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
